sale_fixed_discount/models/sale_order_line.py

```diff
@@ -78,9 +78,14 @@ def _convert_to_tax_base_line_dict(self, **kwargs):
 
         return super()._convert_to_tax_base_line_dict()
 
-    @api.onchange("discount_fixed", "price_unit")
-    def _onchange_discount_fixed(self):
-        self.discount = self._get_discount_from_fixed_discount()
+    @api.depends(
+        "product_id", "product_uom", "product_uom_qty", "discount_fixed", "price_unit"
```

Comment:
```suggestion
       "discount_fixed", "price_unit"
```

ORM will append new fields to existing depends.

Reply:
Right, thanks!